rollup-operator/src/loop-manager.js

```diff
@@ -389,16 +416,98 @@ class LoopManager{
             this.timeouts.NEXT_STATE = 5000;
             // Check I am still the winner
             const deadlineReach = await this._blockDeadline(currentBlock);
-            if (deadlineReach){
+            const currentStateRoot = await this.rollupSynch.getCurrentStateRoot();
+            const currentOnchainHash = await this.rollupSynch.getMiningOnchainHash;
```

Comment:
`const currentOnchainHash = await this.rollupSynch.getMiningOnchainHash;`
- Call a function from `rollupSynch` with `()` even if no argument is passed:
`const currentOnchainHash = await this.rollupSynch.getMiningOnchainHash();`
- just to keep same notation in all files

Reply:
true, sry!